tests/common/test_cgroupapi.py

```diff
@@ -31,3 +36,41 @@ def test_dummy(self):
         api = FileSystemCgroupsApi()
         api.create_agent_cgroups()
         self.assertTrue(True)
+
+
+@skip_if_predicate_false(CGroupConfigurator.enabled, "CGroups not supported in this environment")
+class TestSystemdCgroupsApi(AgentTestCase):
+
+    @skip_if_predicate_false(i_am_root, "Test does not run when normal user")
+    def test_if_extensions_root_slice_is_created(self):
+        SystemdCgroupsApi().create_extension_cgroups_root()
```

Comment:
Can you also write a test which does not directly call SystemdCgroupsApi().X but CgroupsApi().X and see if the valid cgroup methods are being called.